resources/monitoring/templates/prometheus/additionalPrometheusRules.yaml

```diff
@@ -1,19 +1,18 @@
 {{- if .Values.additionalPrometheusRules }}
-{{- range $name, $config := .Values.additionalPrometheusRules | fromYaml }}
----
-apiVersion: monitoring.coreos.com/v1
-kind: PrometheusRule
-metadata:
-  name: {{ template "prometheus-operator.name" $ }}-{{ $name }}
-  namespace: {{ $.Release.Namespace }}
-  labels:
-    app: {{ template "prometheus-operator.name" $ -}}
-{{ include "prometheus-operator.labels" $ | indent 4 -}}
-{{ if $config.additionalLabels }}
-{{ toYaml $config.additionalLabels | trim | indent 4 -}}
-{{- end }}
-spec:
-  groups:
-{{ toYaml $config.groups | indent 4 -}}
-{{- end }}
-{{- end }}
+  {{- range $name, $config := .Values.additionalPrometheusRules | fromYaml }}
```

Comment:
There is discrepancy here from master. https://github.com/kyma-project/kyma/pull/8684 the PR is missing.